docs/getting-started/index.md

```diff
@@ -8,7 +8,29 @@
 
 This page describes how to get started building an application with Warp components.
 
-## 1. Integrate with UnoCSS and Warp
+## 1. Set up fonts
+
+Specify fonts with which to display text in your application. Here's an example for Finn.no:
```

Comment:
I think we can add a note where to specify the fonts? I also think that we can simplify the beginning a little bit, wdyt?
```suggestion
Specify fonts to be used in your application. Here's an example for Finn.no:
```

Reply:
That's something I was unsure about because of some "templates" that were mentioned. Would it be correct to say "add this to your `styles.css` file"? It should be different for each brand, so again I'm confused if that's the place.